CONTRIBUTING.md

```diff
@@ -430,9 +430,9 @@ Getting going
   * Add your remote to the imports in `backend/all/all.go`
   * HTTP based remotes are easiest to maintain if they use rclone's rest module, but if there is a really good go SDK then use that instead.
   * Try to implement as many optional methods as possible as it makes the remote more usable.
-  * Use lib/encoder to make sure we can encode any path name and `rclone info` to help determine the encodings needed
-    * `rclone purge -v TestRemote:rclone-info`
-    * `rclone test info --all --remote-encoding None -vv --write-json remote.json TestRemote:rclone-info`
+  * Use lib/encoder to make sure we can encode any path name and `rclone test info` to help determine the encodings needed
+    * `rclone purge -v TestRemote:rclone-test-info`
+    * `rclone test info --all -vv --write-json remote.json TestRemote:rclone-test-info`
```

Comment:
Why does this remove the `--remote-encoding None` ?

That is needed for the tests to work if you have implemented encoding in the backend. If the backend doesn't implement encoding then it needs to be removed.

Maybe that needs to be spelled out more?

Reply:
All other points I found very clear. I see now that some remotes add this "lib/encoder" via standard flag and use an extra field to run names through the configured encoders. Shorter is better, so closing this (sorry for the noise).

Reply:
No problems and thank you for thinking of sending a doc patch - much appreciated!